docs/_basic/listening_modals.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Listening for view submissions
+lang: en
+slug: view_submissions
+order: 12
+---
+
+<div class="section-content">
+
+If a <a href="https://api.slack.com/reference/block-kit/views">view payload</a> contains any input blocks, you must listen to `view_submission` events to receive their values. To listen to `view_submission` events, you can use the built-in `view()` method. `view()` requires a `callback_id` of type `str` or `re.Pattern`.
+
+You can access the value of the `input` blocks by accessing the `state` object. `state` contains a `values` object that uses the `block_id` and unique `action_id` to store the input values.
+
+Read more about view submissions in our <a href="https://api.slack.com/surfaces/modals/using#interactions">API documentation</a>.
+
+</div>
+
+```python
+# Handle a view_submission event
+​​@app.view("view_b")
+​​def handle_submission(ack, body, client, view):
+​​    # Acknowledge the view_submission event
+​​    ack()
+​​
```

Comment:
@seratch I've managed to lose the validation request you placed here. Could you possibly provide a snippet of what you had in mind so I can fold it in?

Reply:
@misscoded How about having simpler ones such as length-check? Here is an example I recently created: https://github.com/seratch/apidays-workshop-2020/blob/87f623356f90fbdb9f5112e114dc51c157698ac7/python/app.py#L272-L283